kubeval/kubeval.go

```diff
@@ -122,17 +122,19 @@ func validateResource(data []byte, schemaCache map[string]*gojsonschema.Schema,
 		return result, body, nil
 	}
 
-	name, err := getStringAt(body, []string{"metadata", "name"})
-	if err != nil {
-		return result, body, fmt.Errorf("%s: %s", result.FileName, err.Error())
-	}
-	result.ResourceName = name
-
-	namespace, err := getStringAt(body, []string{"metadata", "namespace"})
-	if err != nil {
-		result.ResourceNamespace = "default"
```

Comment:
I don't think this code had any effect in the first place because it would eventually be overwritten.